api/pom.xml

```diff
@@ -17,27 +17,27 @@
 
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"> <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Was there some reason for this header change?

Reply:
Yes, I did a mvn tidy:pom on the pom.xml files to format it according to maven conventions.

(I'm personally a fan of the prior version, but oh well)

Reply:
If you want to promote a position that all projects should apply "tidy:pom" to their pom
files, then it would be fine to do it here.  Otherwise, it just feels like an unnecessary
gratuitous change.

Reply:
Yes, I think all projects should do a tidy:pom. Some did. I'll start a thread about this and/or will do PRs to the other projects.